server/chat-plugins/roomsettings.js

```diff
@@ -349,7 +349,7 @@ exports.commands = {
 			this.addModAction(`${user.name} set modjoin to autoconfirmed.`);
 			this.modlog('MODJOIN', null, 'autoconfirmed');
 		} else if (target in Config.groups || target === 'trusted') {
-			if (room.battle && !user.can('makeroom') && target !== '+') return this.errorReply(`/modjoin - Access denied from setting modjoin past + in battles.`);
+			if (room.battle && !user.can('makeroom') && (target !== '+' || target !== '%')) return this.errorReply(`/modjoin - Access denied from setting modjoin past % in battles.`);
```

Comment:
Should line 277 and or 302 (fuck GH for not letting me comment on them) be `/modjoin %` instead of `+`?

Reply:
Yes, assuming those are the `/inviteonly` implementations.